src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -514,6 +524,18 @@ private int WriteRunToErrorList(Run run, string logFilePath)
             return sarifErrors.Count;
         }
 
+        private void ShowFilteredSuppressionColumn()
+        {
+            // Creating this table source causes the Suppression State column to appear.
+            SarifSuppressedResultsTableDataSource _ = SarifSuppressedResultsTableDataSource.Instance;
+        }
+
+        private void ShowFilteredCategoryColumn()
+        {
+            // Creating this table source causes the Category column to appear.
```

Comment:
>appear [](start = 72, length = 6)

It turns out the Category column is visible by default, and there doesn't seem to be a way to change that. Still investigating.

Reply:
The only way to fix this would be to define a column just like Category (say, BaseLine State), make if invisible by default, and provide a programmatic interface to turn it on. This is consistent with our desire not to overload the Category column.

@mikefan@microsoft.com, would you like me to do that as part of this PR, or file an issue per your comment above?

---
In reply to: [537693891](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r537693891) [](ancestors = 537693891)

Reply:
On second thought, you said elsewhere to file an issue for this, so I did and we'll move on. [https://github.com/microsoft/sarif-visualstudio-extension/issues/315](https://github.com/microsoft/sarif-visualstudio-extension/issues/315), "Decide where to display baseline state."

---
In reply to: [538916975](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538916975) [](ancestors = 538916975,537693891)